thesis-de/content-de.tex

```diff
@@ -361,11 +361,145 @@ \subsubsection{Durchführung der Evaluierung}
 \nl%
 
 \subsection{Frontends}
+\label{subsec:frontends}
+Bevor das Projekt beginnen konnten, musste zusammen mit dem Auftraggeber und
+dem Projektteam, eine Liste an möglichen Frontends erarbeitet werden.
+Zusätzlich zu dieser Liste, wurden verschiedene Anforderungen definiert, welche
+im späteren Verlauf der Planung für die Evaluierung der einzelnen Frontends
+verwendet wird. Diese Liste besteht zum Teil aus Software Lösungen, welche im
+eigenen Unternehmen, bereits erfolgreich eingesetzt worden sind und zum anderen
+von Vorschläge des Auftraggebers. Während der Planung des Frontends, kamen noch
+weitere Frontends zu der Liste hinzu, welche in die Evaluierung mit aufgenommen
+worden sind.
+
+Dies benötigte keine Zustimmung des Auftraggebers, sondern konnte mit dem
+Projektteam besprochen werden.
+
+Folgende Frontends wurden während der Planung, bereits vorab mit dem
+Auftraggeber und dem Porjektteam erarbeitet:
 
 \begin{outline}
   \1 grafana
   \1 graphite
   \1 zabbix-frontend
+  \1 datadog
+  \1 gridster-D3
+  \1 netdata
+\end{outline}
+\mr%
+
+\subsection{Vorbereitung der Evaluierung}
+\label{subsubsec:vorbereiten_der_evaluierung_frontend}
+Damit die Evaluierung der einzelnen Frontends durchgeführt werden konnten,
+mussten zuvor Kriterien zusammen mit dem Porjektteam definiert werden, welche
+die Frontends erfüllen müssen. (Diese sollen später für eine bessere
+Übersichtlichkeit dienen, welche Anforderungen das Frontend haben muss). Bei 
```

Comment:
Keine Klammern. Entweder in einem Satz oder es ist keine wichtige Information. Bitte generell versuchen in Dokumentationen keine ganzen Sätze in Klammern zu formulieren. 
Zudem ist der Satzbau mieserabel :( 
Diese Kriterien, in Form von definierten Anforderungen, sollen für eine besser Übsericht während der Evaluierung dienen.